velox/expression/tests/ArgumentTypeFuzzerTest.cpp

```diff
@@ -73,22 +73,31 @@ TEST_F(ArgumentTypeFuzzerTest, concreteSignature) {
     auto signature = exec::FunctionSignatureBuilder()
                          .returnType("bigint")
                          .argumentType("varchar")
+                         .constantArgumentType("boolean")
                          .argumentType("array(boolean)")
                          .build();
 
-    testFuzzingSuccess(signature, BIGINT(), {VARCHAR(), ARRAY(BOOLEAN())});
+    testFuzzingSuccess(
```

Comment:
The change here should not affect ArgumentTypeFuzzer, hence, I'm surprised to see changes to ArgumentTypeFuzzerTest.cpp. Let's remove these. Let's add tests to velox/exec/tests/FunctionSignatureBuilderTest.cpp instead.

Reply:
Fixed